docs/resources/Guild.md

```diff
@@ -781,21 +773,59 @@ Delete the attached [integration](#DOCS_RESOURCES_GUILD/integration-object) obje
 
 Sync an integration. Requires the `MANAGE_GUILD` permission. Returns a 204 empty response on success.
 
-## Get Guild Widget % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/widget
+## Get Guild Widget Settings % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/widget
 
 Returns the [guild widget](#DOCS_RESOURCES_GUILD/guild-widget-object) object. Requires the `MANAGE_GUILD` permission.
 
-## Get Guild Embed % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/embed
+###### Example Get Guild Widget Settings
 
-Same as above, but this endpoint is deprecated.
+```json
+{
+    "enabled": true,
+    "channel_id": "639513359995470208"
+}
+```
```

Comment:
duplicate of https://discord.com/developers/docs/resources/guild#guild-widget-object-example-guild-widget